front_end/src/public/js/bootstrap.js

```diff
@@ -0,0 +1,2363 @@
+/*!
+ * Bootstrap v3.3.5 (http://getbootstrap.com)
```

Comment:
there's bower for client side packages.
I suggest we either use bower or we "vendor" off the depedencies.